Resources/Prototypes/_CP14/Entities/Mobs/NPC/cackle.yml

```
@@ -0,0 +1,134 @@
+- type: entity
+  parent: CP14SimpleMobBase
+  id: CP14MobMonsterCackle
+  name: Cackle
+  description: A monster that hunts for blood as a source of sustenance.
+  categories: [ ForkFiltered ]
+  components:
+  - type: HTN
+    rootTask:
+      task: SimpleRangedHostileCompound
+    blackboard:
+      NavSmash: !type:Bool
+        true
+      VisionRadius: !type:Single
+        16
+      AggroVisionRadius: !type:Single
+        6
+  - type: Sprite
+    drawdepth: Mobs
+    sprite: _CP14/Mobs/Monster/cackle.rsi
+    layers:
+      - map: [ "enum.DamageStateVisualLayers.Base" ]
+        state: live
+      - map: [ "enum.DamageStateVisualLayers.BaseUnshaded" ] #used cause i dont know if there are any more applicable ones
+        state: live_spikes
```

```suggestion
        state: live_spikes
        shader: unshaded
```
If you want the spikes to glow in the dark, add ‘shader: unshaded’.